event-sourcing/event-repository/event-repository-jdbc/src/test/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventstream/EventStreamJdbcRepositoryFactoryTest.java

```diff
@@ -52,4 +58,12 @@ public void shouldProduceEventStreamJdbcRepository() throws Exception {
         final Logger loggerField = getValueOfField(eventStreamJdbcRepository, "logger", Logger.class);
         assertThat(loggerField, is(notNullValue()));
     }
+//
+//    @Test
+//    public void shouldProduceEventStreamJdbcRepositoryWithDataSource() throws Exception {
+//
+//        final EventStreamJdbcRepository eventStreamJdbcRepository = eventStreamJdbcRepositoryFactory.eventStreamJdbcRepository(datasource);
+//
+//        assertThat(eventStreamJdbcRepository, is(notNullValue()));
+//    }
```

Comment:
This commented out test should be reinstated or removed

Reply:
removed